### PR TITLE
Tweets search interface

### PR DESCRIPTION
**Pull request by annarieger**
This PR introduces the heatmap hooked up to the API `/tweets/search/` avaliable under http://54.158.101.33:8080/bopws/swagger/#!/default/search

The important changes:
- allow search on given keyword (`q.text`), time range (`q.time`) and bounding box (`q.geo`)
- search can also be performed, if no keyword was provided (`q.text="*"`)
- big code refactoring due to the new API response structure
  - service `GeoNodeSolr` won't be used any more
  - services `HeatMapSourceGenerator` and `Map` could be significantly simplified since we have only 3 search parameters (keyword, time and geo bbox)

TODO:
- We currently used only three parameters mentioned above and let the API use the default values for all other parameters. This should be discussed, to find way to combine these additional parameters with the mandatorys meaningful.
- To get the heatmap displayed we currently use the static parameter `a.hm.limit = 10`. This value should probably also get dynamic.

Please review.
/cc @marcjansen, @dnlkoch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/angularjs-slider/2.13.0/rzslider.min.js"></script>
     <script src="./assets/lib/proj4js/proj4.js"></script>
     <script src="./config/3857.js"></script>
-    <!-- <script src="./assets/lib/ol3/ol-debug.js"></script> -->
-    <script src="./assets/lib/ol3/ol.js"></script>
+    <script src="./assets/lib/ol3/ol-debug.js"></script>
```
Comment (weskamm) [inline index.html]: when using ol-debug, wouldn't it be nicer to have an index-dev.html?
Or was this just for testing purposes and forgotten?

**Comment by weskamm** [comment]
Just added a minor comment. Looking ok to me so far, but my insight into the API is limited, so another pair of eyes on this would make sense

File: index.html (hunk 1 above)
Comment (annarieger) [inline index.html]: That was forgotten :innocent:

**Comment by annarieger** [comment]
Thanks, I'll merge this PR now..

**Pull request merged**